Mathlib/LinearAlgebra/Finsupp/LinearCombination.lean

```diff
@@ -59,21 +67,6 @@ theorem linearCombination_single (c : R) (a : α) :
 theorem linearCombination_zero_apply (x : α →₀ R) : (linearCombination R (0 : α → M)) x = 0 := by
   simp [linearCombination_apply]
 
-variable (α M)
-
-@[simp]
-theorem linearCombination_zero : linearCombination R (0 : α → M) = 0 :=
```

Comment:
This is the line I was arguing sopuld not have moved

Reply:
OMG, I'll open a PR at once to restore that one and the one after at this place, now I see your point.

Reply:
#22713